src/settings.cpp

```diff
@@ -268,6 +267,9 @@ void read_settings_xml()
     }
   }
 
+  // Check for user meshes and allocate
+  read_meshes(root);
```

Comment:
Since we've now moved this call up to here, let's remove the call that happens later on in this function so we aren't doing this twice.